setup.py

```diff
@@ -19,31 +19,45 @@ def _post():
     nltk.download('stopwords')
     nltk.download('punkt')
 
+
 class PostInstallCommand(install):
     """Post-installation for installation mode."""
     def run(self):
         install.run(self)
         _post()
 
+
 class PostDevelopCommand(develop):
     """Post-installation for development mode."""
     def run(self):
         develop.run(self)
         _post()
 
 
+# pypi does not render markdown.
+# https://stackoverflow.com/a/26737672/19741
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+except(IOError, ImportError):
+    long_description = open('README.md').read()
```

Comment:
There is not any other use that I know of.

Reasonable, but I think it would be good to hit the breaks even before uploading something that may need change, so as to facilitate the life of the release manage (who probably already is dealing with other stuff). If you don't think it makes any sense, this should be good then.